### ISSUE TITLE
[Request] Search plugins for anime torrent index sites (tokyotosho, shanaproject)

### ISSUE THREAD

**Issue by M1suzu** [qbittorrent/search-plugins#13]
Both https://www.tokyotosho.info/ and https://www.shanaproject.com/ are one of the best anime index websites, can somebody write plugin for them, I tried to do that but my python knowledge is suck and all I can do is modify or update existing plugin.

**Comment by Shachar2like** [comment]
On the same issue, I'm going over on my free time learning python.

Can you give me a quick overview on the command scripts that's provided?
I mean the script comes with ready made "commands"
Where do I get more information or documentation on them?
Or what specifically do I need to know?

When you do change a script to search a website do you "just" change the script to point to the new website?
How do you debug it?
For example I've seen in the script that if it's a website that allows anyone to go and search then you wouldn't have a problem but if it's a website that requires a cookie or a free registration then you might need to program a different (crawler? it had a different word that I don't remember off the top of my head).
The script also mention that you (crewel?) the website and put it in your "definition"
I assume that's where you store the data 'results' from the website, is that a file that you save on the computer or simply to memory?

In short, is there some more technical explanation of how to program/modify a search engine?

**Comment by Pireo** [comment]
tokyotosho: https://raw.githubusercontent.com/DoumanAsh/qBittorrent-Search-Plugins/master/tokyotoshokan/tokyotoshokan.py

**Comment by Chocobo1** [comment]
>tokyotosho: https://raw.githubusercontent.com/DoumanAsh/qBittorrent-Search-Plugins/master/tokyotoshokan/tokyotoshokan.py

@DoumanAsh
You mind adding an entry in the wiki?
https://github.com/qbittorrent/search-plugins/wiki/Unofficial-search-plugins

**Comment by DoumanAsh** [comment]
I'll check if it works today evening or update plugin and add entry

**Comment by sledgehammer999** [comment]
Question is, why don't you submit the plugins in this repo?

**Comment by sledgehammer999** [comment]
>I can submit. It is just the plugin was written long ago and for some time I didn't use it even myself.

Just to be clear, you should submit if you're also willing to support it in the future. Otherwise, just leave it as an unofficial entry.

**Comment by Pireo** [comment]
@M1suzu now that tokyo toshokan has been updated and reposted, and there are aldready meany plugins about anime, you can close this issue.
